services/dy-jupyter/handlers/input_retriever.py

```diff
@@ -60,78 +53,121 @@ def _no_relative_path_zip(members: zipfile.ZipFile):
             continue
         yield zipinfo.filename
 
+async def get_time_wrapped(port):
+    logger.info("transfer started for %s", port.key)
+    start_time = time.perf_counter()
+    ret = await port.get()
+    elapsed_time = time.perf_counter() - start_time
+    logger.info("transfer completed in %ss", elapsed_time)
+    if isinstance(ret, Path):

```

Comment:
I guess ``ret`` can be ``Path`` or ``None``. Is there something else? therefore it is more clear
```python
if ret:
 ...
```

Reply:
it can be something else, such as an int, float, string...